addons/mail/static/src/core/common/thread_model.js

```diff
@@ -670,7 +672,7 @@ export class Thread extends Record {
     }
 
     /** @param {{after: Number, before: Number}} */
-    async fetchMessages({ after, before } = {}) {
+    async fetchMessages({ after, around, before, limit } = {}) {
```

Comment:
`limit` params is useless: it's `2x FETCH_LIMIT` when `around` is provided, otherwise `FETCH_LIMIT`